adam-core/src/test/scala/org/bdgenomics/adam/rdd/ADAMContextSuite.scala

```diff
@@ -760,4 +759,91 @@ class ADAMContextSuite extends ADAMFunSuite {
     assert(secondPg.head.getCommandLine === "\"myProg 456\"")
     assert(secondPg.head.getVersion === "1.0.0")
   }
+
+  sparkTest("load variant contexts from dataframe") {
```

Comment:
@fnothaft This test (and it's equivalent form where we reload using the `loadVariantContexts(path: String)` method) don't pass against my PR or master, which surprised me. Any ideas why?

Reply:
I'll take a look into the failure.

Reply:
Not sure why this is failing; I took the test output and pasted the dump of all of the records in each array and they have the same textual values. You could either be getting bitten by some odd floating point comparison bug, or perhaps the Array== comparator does something wonky.